src/ophys_etl/workflows/pipeline_modules/segmentation.py

```diff
@@ -50,7 +55,8 @@ def inputs(self) -> Dict:
                     self.ophys_experiment.movie_frame_rate_hz
                 )
             },
-            "postprocess_args": {}
+            "postprocess_args": {},
+            "output_json": self.output_metadata_path
```

Comment:
this is passed separately to the CLI. Do not need to pass it here. See `pipeline_module.py`

Reply:
the validation doesn't pass without it added since a mm.preload references it. 
https://github.com/AllenInstitute/ophys_etl_pipelines/blob/989374d81453c209e038cdf1ad8531d389354b0e/src/ophys_etl/modules/segment_postprocess/schemas.py#L55

Reply:
I believe it is getting `output_json` that is passed in from the command line, but when you call `SegmentPostprocessSchema().load()` it is not getting passed. You can pass it to the constructor of `SegmentPostprocessSchema()` for testing purposes maybe.

Reply:
I'm not clear how you think this should be designed. Maybe you can do it.

Reply:
Either way, if it's defined explicitly in the CLI, it would overwrite what's set there.

Reply:
ok i'm ok with it

Reply:
I verified that CLI would overwrite the input_json. I tested this out by running a module with output_json added a param in the input_json and in the cli. 
I printed the args and it printed the cli param.